app-shells/bash/bash-5.1.8.recipe

```diff
@@ -63,7 +63,7 @@ PATCH()
 BUILD()
 {
 	CPPFLAGS="-DUSE_MKTEMP=1 -DUSE_MKSTEMP=1" runConfigure ./configure \
-		--without-bash-malloc --with-installed-readline
+		--without-bash-malloc --disable-rpath --enable-disabled-builtins --with-installed-readline
```

Comment:
What does the `--enable-disabled-builtins` do?

Reply:
> --enable-disabled-builtins
> Allow builtin commands to be invoked via ‘builtin xxx’ even after xxx has been disabled using ‘enable -n xxx’. See Bash Builtins, for details of the builtin and enable builtin commands.

Sounds quite useful to me, why remove it?